src/main/java/org/gridsuite/study/server/networkmodificationtree/dto/RootNode.java

```diff
@@ -38,6 +40,8 @@
 
     BuildStatus buildStatus;
 
+    Set<UUID> modificationsToExclude = new HashSet<>();
```

Comment:
Since no modification are associated to Root node, is this attribute useful ?

Reply:
Same